src/components/datepicker/DatepickerTable.vue

```diff
@@ -53,6 +57,38 @@
                     index++
                 }
                 return visibleDayNames
+            },
+
+            hasEvents() {
+                return this.events && this.events.length
+            },
+
+            /*
+            * Return array of all events in the specified month
+            */
+            eventsInThisMonth() {
+                if (!this.events) return []
+
+                const monthEvents = []
+
+                for (let i = 0; i < this.events.length; i++) {
+                    let event = this.events[i]
+
+                    if (!event.hasOwnProperty('date')) {
+                        event = { date: event }
+                    }
+                    if (!event.hasOwnProperty('type')) {
+                        event.type = 'is-primary'
+                    }
+                    if (Object.prototype.toString.call(event.date) !== '[object Date]') {
```

Comment:
Datepicker accepts only `Date` as `v-model`, `min`, `max` and `unselectable-dates`.
I think it's best if we manage only `Date` at the moment.